projects/origin-release/ipi_deprovision/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
```

Comment:
This file should be somewhere under `core-services`, or even better, in the `ci-tools` repo.

Reply:
https://github.com/openshift/ci-tools/pull/26

Reply:
https://github.com/openshift/ci-tools/pull/26